dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -134,5 +135,12 @@ object GlobalRum {
         activeContext.set(newContext)
     }
 
+    // For Tests only
+    @Suppress("unused")
+    @JvmStatic
+    private fun resetSession() {
```

Comment:
It is a bit weird to have this...I would rather call "resetSession" through reflection on `GlobalRum.get()` from the Shopist level.

Reply:
Again, calling it on GlobalRum.get() makes it more fragile to changes.